draft-ietf-dnssd-update-lease.xml

```diff
@@ -232,10 +235,10 @@ KEY-LEASE        u_int32_t    optional desired (or granted)
 	update or a Refresh, remain active, MUST send a Refresh message before the lease elapses, or else the records
 	will be removed by the server.</t>
 
-	<t>Requestors SHOULD Refresh resource records after approximately 75% of the original
-	lease has elapsed. This interval MUST be adjusted randomly by +/-10% so that after a
-	network restart, subsequent Renews do not remain synchronized across devices affected
-	by the power failure.</t>
+	<t>In order to prevent records expiring, requestors MUST Refresh resource records after
+	approximately 75% of the original lease has elapsed. This interval MUST be adjusted
+	randomly by +/-10% so that after a network restart, subsequent Renews do not remain
+	synchronized across devices affected by the power failure.</t>
```

Comment:
Same comment as on another commit: would be good to be clear on whether this means 65% to 85%, or (75% +/- 10% of that).